sort/gnome_sort

```diff
@@ -0,0 +1,42 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<math.h>
+
+void main()
+{
+  int arr[]={-5,6,-7,10,4,1};
```

Comment:
Could you dynamically allocate the array ? @Budcoderr

Reply:
Shall i use linked list or just ask the user how many elements he wants to enter in the array and then allocate it?

Reply:
Go with the array

Reply:
i have updated it with array